COLLECTOR_VERSION

```diff
@@ -1 +1 @@
-3.19.x-70-gf3a66fa3a5
+3.19.x-69-g4fb18e548a
```

Comment:
The collector version was updated to one that is capable of receiving the configuration. This is just for testing purposes and this change will be undone before merging.